Description: Remove unused gnome-keyring from build-deps.
Author: Fabio Fantoni <fantonifabio@tiscali.it>
Forwarded: not-yet
Last-Update: 2017-08-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -81,7 +81,7 @@
                                libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_MIN_VERSION
                                gobject-introspection-1.0 >= $GOBJECT_INTROSPECTION_MIN_VERSION
                                polkit-agent-1 >= $POLKIT_MIN_VERSION xfixes
-                               gnome-keyring-1 atk-bridge-2.0)
+                               atk-bridge-2.0)
 
 PKG_CHECK_MODULES(CINNAMON_NM, libnm-glib libnm-util,
                   [have_nm=true], [have_nm=false])
